Solidner/Views/Onboarding/BabyBirthDateView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BabyBirthDateView.swift
+//  Solidner
+//
+//  Created by 황지우2 on 2023/11/13.
+//
+
+import SwiftUI
+
+struct BabyBirthDateView: View {
+    let bigTitleColor = Color(#colorLiteral(red: 0.0834152922, green: 0.0834152922, blue: 0.0834152922, alpha: 1))
+    let smallTitleColor = Color(#colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)).opacity(0.6)
+    let cakeLabelFontSize = 32.0
+    let onboardingTitlesTopPadding = 10.0
+    let datePickerTopPadding = 70.0
+    @State private var babyName = "찍무"
+    @State private var babyBirthDate = Date()
+    var body: some View {
+        VStack(spacing: 0) {
+           BackButtonHeader()
+            viewBody()
+        }
+    }
+    func viewBody() -> some View {
```

Comment:
이런 function들도 struct 안쪽에 넣은 거 보면 재사용할 일이 없나요?
그렇다면 private ㄱ
```suggestion
    private func viewBody() -> some View {
```